x-pack/legacy/plugins/siem/server/lib/detection_engine/types.ts

```diff
@@ -49,7 +51,7 @@ export interface RuleAlertParams {
   threat: ThreatParams[] | undefined | null;
   type: 'query' | 'saved_query';
   version: number;
-  throttle?: string;
+  throttle: string | undefined | null;
```

Comment:
Might be safer to type this from `string | undefined | null` to `'no_actions' | 'rule' | '1h' | '1d' | '7d' | undefined | null`, since it looks like throttle can only have those values, and not just any string.